packages/expo-splash-screen/README.md

```diff
@@ -190,7 +190,7 @@ const styles = StyleSheet.create({
 
 ## 💻 Installation in managed Expo projects
 
-[Managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects are not supported by this unimodule at this moment. Instead, you should use the `SplashScreen` module that is within the `expo` package (`import { SplashScreen } from 'expo'`).
+[Managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects uses the older `SplashScreen` module within the `expo` package (`import { SplashScreen } from 'expo'`).
```

Comment:
```suggestion
[Managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects use the older `SplashScreen` module within the `expo` package (`import { SplashScreen } from 'expo'`).
```

Reply:
[expo-splash-screen] fix grammar in readme